$MirOS: ports/print/cups/patches/patch-scheduler_log_c,v 1.1 2006/11/03 12:33:26 bsiegert Exp $
--- scheduler/log.c.orig	Sat Jul 12 00:48:26 2008
+++ scheduler/log.c	Tue Oct 21 22:56:23 2008
@@ -93,7 +93,7 @@ cupsdGetDateTime(time_t t)		/* I - Time 
     date = localtime(&t);
 
     snprintf(s, sizeof(s), "[%02d/%s/%04d:%02d:%02d:%02d %+03ld%02ld]",
-	     date->tm_mday, months[date->tm_mon], 1900 + date->tm_year,
+	     date->tm_mday, months[date->tm_mon], 1900 + (int)date->tm_year,
 	     date->tm_hour, date->tm_min, date->tm_sec,
 #ifdef HAVE_TM_GMTOFF
              date->tm_gmtoff / 3600, (date->tm_gmtoff / 60) % 60);
